src/java.base/share/classes/sun/security/pkcs12/PKCS12KeyStore.java

```diff
@@ -661,30 +661,23 @@ private void setKeyEntry(String alias, Key key,
                 keyEntry.date = new Date();
 
                 // Encode secret key in a PKCS#8
-                DerOutputStream pkcs8 = new DerOutputStream();
                 DerOutputStream secretKeyInfo = new DerOutputStream();
```

Comment:
Same, using default constructor here and we write encodedKey into it at line 670 below.

Reply:
Same as above.